symless/synergy-core
# Add Git SHA to about screen and `--version`

https://symless.atlassian.net/browse/S1-1708

To test:
1. Set env var `GIT_SHA` to `123456789`
2. Configure
3. Check GUI about screen
4. Expect: "(12345678)" after version
5. Run: `synergys --version`
6. Expect: "(12345678)" after version
